src/components/InformationBoxComponent/InformationBoxComponent.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const InformationBox = ({ title, imagePath, description }) => {
+  return (
+    <div style={styles.container}>
+      <div style={styles.iconContainer}>
+        <img src={imagePath} alt={`${title} icon`} style={styles.icon} />
+      </div>
+      <h3 style={styles.title}>{title}</h3>
+      <p style={styles.description}>{description}</p>
+    </div>
+  );
+};
+
+const styles = {
```

Comment:
You can move all this to `src/components/InformationBoxComponent/InformationBox.module.css` file       The style you write in this is same as what's been used in css, `.classname`

then do `import styles from ./InformationBox.module.css` in this file (`src/components/InformationBoxComponent/InformationBoxComponent.jsx`)